rust/BUILD.md

```diff
@@ -31,6 +31,12 @@ instead of `onnxruntime/rust/target`.
     CARGO_TARGET_DIR=build/rust cargo test --manifest-path rust/Cargo.toml --features model-fetching
 ```
 
+### cargo test both crates while specifying the absolute path to the OnnxRuntime shared library.
+
+```sh
+    RUST_ONNXRUNTIME_LIBRARY_PATH=<absolute path to shared library> CARGO_TARGET_DIR=build/rust cargo test --manifest-path rust/Cargo.toml --features model-fetching
+```
+
```

Comment:
Ah, *_PATH made me think folder vs the actual filename. Since it should always be called 'onnxruntime.dll' I didn't think to give the name of the library itself.

Up to you if you think it should let you override the library filename, since it does make the path more platform specific. But I'd recommend adding an example just so remove all doubt.

Reply:
Another idea, as you mentioned running 'cargo c' and seeing where it goes, could this process set the environment variable to where it was put if the variable isn't already set? Or is that a bad idea?

Ideally it'd "just work", but it's up to you. This would work across all OS's hopefully.

Reply:
Added libonnxruntime.so as an example.